### PR TITLE
readme cn

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: README-cn.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+|     `- storage2
+`-- logs
+     |- meta0
+     |- meta1
+     |- meta2
+     |- storage0
+     |- storage1
+     |- storage2
+     `- graph
+```
```
Comment (yixinglu) [inline README-cn.md]: 上面的这里更新过啦，你看最新的 readme

**Comment by Amber1990Zhang** [comment]
> Chinese readme filename: README-cn.md vs README-zh.md, which is better ? I prefer the later.

According to ISO 3166 alpha-2, cn is short for China. I think cn is more international.

**Comment by yixinglu** [comment]
cn is for china but zh is for chinese

**Comment by yixinglu** [comment]
Add link to Chinese readme in English version

**Pull request merged**